src/revivalpmmp/pureentities/PureEntities.php

```diff
@@ -116,7 +119,7 @@ public function onLoad(){
             Mule::class,
             //ElderGuardian::class,
             //Guardian::class,
-	        //Bat::class,
+            //Bat::class,
```

Comment:
not sure why people still having commented out lines. This is VCS, we can find it in history. Just wondering.